src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -17,31 +17,52 @@ import {
   IResourceDefinition,
 } from '@floip/flow-runner/src/domain/IResourceResolver'
 import Vue from 'vue'
-import {defaults, find, max} from 'lodash'
+import {defaults, find, max, filter, first, get} from 'lodash'
 
 export const BLOCK_TYPE = 'MobilePrimitives\\SelectOneResponse'
 
 import { someItemsHaveValue, allItemsHaveValue, twoItemsBlank } from '../utils/listBuilder'
 
+interface IInflatedChoicesInterface {
+  exit: IBlockExit,
+  resources: IResourceDefinition
+}
+
 export const getters: GetterTree<IFlowsState, IRootState> = {
   inflatedChoices: (state, getters, rootState, rootGetters): object => {
     const currentBlock = rootGetters['builder/activeBlock']
-    let choices: {[key: string]: IResourceDefinition} = {}
-    return Object.keys(currentBlock.config.choices).reduce((memo, choiceKey): {[key: string]: IResourceDefinition} => {
-      memo[choiceKey] = rootGetters['flow/resourcesByUuid'][currentBlock.config.choices[choiceKey]]
+    let choices: { [key: string]: IInflatedChoicesInterface } = {}
+
+    return Object.keys(currentBlock.config.choices).reduce((memo, choiceKey): {[key: string]: IInflatedChoicesInterface} => {
+      const resourceUuid = currentBlock.config.choices[choiceKey]
+      memo[choiceKey] = {
+        exit: getters.blockExitFromResourceUuid(resourceUuid),
+        resources: rootGetters['flow/resourcesByUuid'][resourceUuid]
+      }
       return memo
     }, choices)
   },
+  blockExitFromResourceUuid: (state, getters, rootState, rootGetters) => (resourceUuid: string): IBlockExit => {
+    const currentBlock = rootGetters['builder/activeBlock']
+    return first(filter(currentBlock.exits, {
+      config: {
+        choiceResource: resourceUuid
```

Comment:
So we no longer use the exit label to identify the resource?

Reply:
hum, ... when thinking, we can re-use `exit.label`. thanks for this ;), changing ...

Reply:
changed !